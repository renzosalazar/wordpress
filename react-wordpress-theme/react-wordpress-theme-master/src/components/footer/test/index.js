/**
 * Test for footer component.
 *
 * @package
 */

/**
 * External dependencies
 */
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import Footer from '../index';

describe( 'Test Footer Component', () => {
	it( 'Should match Footer snapshot and render correctly', () => {
		const data = {
			footer: {
				copyrightText: '',
				footerMenuItems: '',
				socialLinks: [],
				sidebarTwo: '',
				sidebarOne: '',
			},
		};
		const loading = false;
		const errorMsg = true;

		const component = renderer.create(
			<Footer footerData={ { data, loading, errorMsg } } />
		);

		const tree = component.toJSON();

		expect( tree ).toMatchSnapshot();
	} );
} );
