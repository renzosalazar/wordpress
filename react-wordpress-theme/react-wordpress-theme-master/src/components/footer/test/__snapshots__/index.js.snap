// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer Component Should match Footer snapshot and render correctly 1`] = `
<div
  className="footer"
>
  <div
    className="wrapper"
  >
    <div
      className="footer__top"
    >
      
      
      
    </div>
    <div
      className="footer__bottom"
    >
      Â© 2020 CODEYTEK ACADEMY.
      
    </div>
  </div>
</div>
`;
